development/tools/build-cleaners.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu
+
+# shellcheck disable=SC2046
+readonly TOOLS_DIR="${PWD}/development/tools"
+cd "$TOOLS_DIR"
```

Comment:
`cd "${TOOLS_DIR}"` with the parenthesis should enable you to remove above shellcheck disable.